notifications/push/fcm/notification.go

```diff
@@ -25,30 +27,35 @@ func NewNotification(key string) NotificationConstructor {
 		client := fcm.NewFcmClient(key).
 			SetDelayWhileIdle(true).
 			SetContentAvailable(true).
+			SetPriority(fcm.Priority_HIGH). // Message needs to be marked as high-priority so that background task in an Android's recipient device can be invoked (https://github.com/invertase/react-native-firebase/blob/d13f0af53f1c8f20db8bc8d4b6f8c6d210e108b9/android/src/main/java/io/invertase/firebase/messaging/RNFirebaseMessagingService.java#L56)
 			SetTimeToLive(fcm.MAX_TTL)
 
 		return &Notification{client}
 	}
 }
 
-// Send send to the tokens list.
-func (n *Notification) Send(body string, payload fcm.NotificationPayload, tokens ...string) error {
-	data := map[string]string{
-		"msg": body,
-	}
+// Send sends a push notification to the tokens list.
+func (n *Notification) Send(dataPayloadJSON string, tokens ...string) error {
+	log.Debug("calling Firebase Send", "dataPayloadJSON", dataPayloadJSON)
 
-	if payload.Title == "" {
-		payload.Title = "Status"
-	}
-	if payload.Body == "" {
-		payload.Body = "You have a new message"
+	var dataPayload map[string]string
+	err := json.Unmarshal([]byte(dataPayloadJSON), &dataPayload)
+	if err != nil {
+		return err
 	}
 
-	fmt.Println(payload.Title, payload.Body)
+	n.client.NewFcmRegIdsMsg(tokens, dataPayload)
+	resp, err := n.client.Send()
+	if err != nil {
+		log.Error("error calling Firebase Send", "err", err.Error())
+		return err
+	}
 
-	n.client.NewFcmRegIdsMsg(tokens, data)
-	n.client.SetNotificationPayload(&payload)
-	_, err := n.client.Send()
+	if !resp.Ok {
+		err = fmt.Errorf("failed to send notification: code=%d err=%s", resp.StatusCode, resp.Err)
+		resp.PrintResults()
```

Comment:
Is it possible to wrap it in `log.Error`?

Reply:
It's not, but with the new FCM form we don't actually need this anymore as we surface the error message in the error object, so I'll remove this line.